backend/src/main/java/de/sambam/gamenighttracker/controller/PlayerController.java

```diff
@@ -0,0 +1,26 @@
+package de.sambam.gamenighttracker.controller;
+
+import de.sambam.gamenighttracker.model.Player;
+import de.sambam.gamenighttracker.service.PlayerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("players")
+public class PlayerController {
+
+    private final PlayerService playerService;
+
+    @Autowired
+    public PlayerController(PlayerService playerService) {
+        this.playerService = playerService;
+    }
+
+    @PostMapping
+    public Player addNewPlayer(@RequestBody Player newPlayer) {
+        playerService.addPlayer(newPlayer);
+        return newPlayer;
```

Comment:
Return the Player object you get from the service. Say you use mongodb and on adding a player it gets an id, but you never return a player with that id. 

Maybe introduce a dto for adding a new player, i don't think all fields from `Player` are filled in the frontend on creation.